### PR TITLE
Stripe accounts update

### PR DESCRIPTION
**Pull request by pushplay**
Updated accounts:
- added missing properties
- fixed metadata typing on update and create
- `type=standard` can no longer be done through the REST API

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/api/accounts/object?lang=node
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
Would like for another owner to review this before merging

**Comment by armanio123** [comment]
Hang in there, I'll merge this PR tomorrow morning if we don't get any review.

**Comment by pushplay** [comment]
Thanks.

**Review by armanio123 — APPROVED** [review]

**Pull request merged**
